BotProject/Templates/CSharp/Scripts/create.ps1

```diff
@@ -91,7 +91,7 @@ Write-Host "> Creating resource group ..."
 
 # Deploy Azure services
 Write-Host "> Validating Azure deployment ..."
-$validation = az group deployment validate `
+$validation = az deployment group validate `
```

Comment:
Can you also add a version guard on top of the file? maybe pick the version az-cli switch to this new syntax? @zidaneymar any suggestions on the version here?

Reply:
@boydc2014 thanks for your reply!

As follows, it seems that az CLI version (az--version) cannot be parsed programmatically...
https://github.com/Azure/azure-cli/issues/11425

Meanwhile, we can get the version programmatically with the following new features, az version command. However, it is still a preview feature.
https://github.com/Azure/azure-cli/pull/11680

Do you have any good suggestions about this?